css/styles.css

```diff
@@ -233,6 +233,15 @@ img.animated {
 .attribution p{
   font-size: 1em;
 }
+.page-tear {
+  position: absolute;
+  bottom: 0;
+  background: url("../assets/bg-texture-white-lighter.png") bottom right repeat, url("../assets/section-bottom-tear-black.png") bottom no-repeat;
```

Comment:
@jereford play around with the positioning `bottom right`. Try `bottom` or `top left` etc to see if you prefer another style. The images cascade so the first one is on top with their respective positioning.